src/Spec2-CommonWidgets/SpFilteringListPresenter.class.st

```diff
@@ -265,6 +265,12 @@ SpFilteringListPresenter >> unfilteredItems [
 	^ unfilteredItems
 ]
 
+{ #category : 'accessing' }
+SpFilteringListPresenter >> unfilteredItems: col [
```

Comment:
`col` is not a meaningful parameter name.
could be `someItems`

Reply:
Agreeee!!! Changed! 💯